pkg/core/util/hive/kernelcr.go

```diff
@@ -420,7 +420,14 @@ func (pluginHandler *PluginHandler) PluginserviceStart(driverConfig *config.Driv
 			serviceConfig["log"] = driverConfig.CoreConfig.Log
 			serviceConfig["env"] = driverConfig.CoreConfig.Env
 			go pluginHandler.handle_errors(driverConfig)
-			go pluginHandler.handle_dataflowstat(driverConfig, mod, vault)
+			currentTokenName := "config_token_pluginany"
+			pluginConfig["tokenptr"] = driverConfig.CoreConfig.TokenCache.GetToken(currentTokenName)
+			_, statmod, v, err := eUtils.InitVaultModForPlugin(pluginConfig, currentTokenName, driverConfig.CoreConfig.Log)
```

Comment:
Needs a...
if v != nil {
		defer v.Close()
	}

after the err check of coure.